### PR TITLE
Add fastlane screenshot support and UI tests

### PR DESCRIPTION
**Pull request by hellais**
This branch is now working and ready for review.

* [x] Come up with a way of creating testing fixtures (see: https://medium.com/@pfandrade_/managing-ios-ui-testing-fixtures-d1f9d11154fd)
* [x] Add support for taking the screenshots mentioned by @agrabeli 
* [x] Add support for embedding screenshot inside of phone (see: https://docs.fastlane.tools/getting-started/ios/screenshots/#put-your-screenshots-into-device-frames)
* [x] Update CI to run this as part of the build

### CODE REVIEW

File: fastlane/Snapfile [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#   "iPhone 8",
+#   "iPhone 8 Plus",
+#   "iPhone SE",
+#   "iPhone X",
+#   "iPad Pro (12.9-inch)",
+#   "iPad Pro (9.7-inch)",
+#   "Apple TV 1080p"
+# ])
+
+# XXX do we want more devices?
```
Comment (lorenzoPrimi) [inline fastlane/Snapfile]: for sure we want iPad. I don't think iPhone 8 is useful. Maybe iPhone 8 Plus could be

File: fastlane/Snapfile [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+devices([
+    "iPhone 8",
+    "iPhone X"
+])
+
+# XXX we should generate this from ooni/translations
+languages([
```
Comment (lorenzoPrimi) [inline fastlane/Snapfile]: well it's in a different repo.
for now the languages are "ar ca de el es fa fr hi it pt_BR tr sq zh_TW"

Comment (hellais) [reply fastlane/Snapfile]: Yeah. When you run `update_language_ios.sh` it should write to this file and also to the `fastlane/en/title.strings` for the text to include in the screenshots.

Comment (lorenzoPrimi) [reply fastlane/Snapfile]: makes sense. let's do it after we merge this PR maybe

**Pull request merged**